homeassistant/components/neato/config_flow.py

```diff
@@ -2,16 +2,15 @@
 
 import logging
 
+import pybotvac
```

Comment:
I had to use this instead of 
```python
from pybotvac.exceptions import NeatoLoginException, NeatoRobotException
from pybotvac import Account, Neato, Vorwerk
```
because it led to failed tests. If someone knows a better solution, I would change the line.

Reply:
If we keep the previous import style we need to change the paths in the patching of the tests.

Reply:
Thank you very much